### PR TITLE
Added Role entity

### PR DESCRIPTION
**Pull request by anavduo**
I added RoleEntity with its attributes and relations

### CODE REVIEW

**Comment by anavduo** [comment]
Already added also the repository

File: src/main/java/com/alkemy/ong/entity/RoleEntity.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.alkemy.ong.entity;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.GenericGenerator;
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Table(name = "role")
```
Comment (juan1977lopez) [inline src/main/java/com/alkemy/ong/entity/RoleEntity.java:9]: La tabla se deberia llamar "roles"

**Review by juan1977lopez — COMMENTED** [review]
Favor revisar comentarios

File: src/main/java/com/alkemy/ong/entity/RoleEntity.java (hunk 1 above)
Comment (anavduo) [reply src/main/java/com/alkemy/ong/entity/RoleEntity.java]: Table named "roles"

**Review by juan1977lopez — APPROVED** [review]
Se aprueba PR

**Pull request merged**